specification/dartLangSpec.tex

```diff
@@ -11267,9 +11303,20 @@ \subsubsection{Ordinary Invocation}
 (\ref{privacy})
 instance member named $m$, unless either:
```

Comment:
Does `X & Foo` have an accessible instance member `foo` if `Foo` has?
(I admit I don't know how we specify the member lookup on type variables).

That is, do we need the *associated bound type* to have the member, or can we say it directly on the type variable type?

Reply:
I believe we have no spec language on the existence of members apart from the notion of 'the interface of a class', which is typically treated as a set of member signatures.

Added a rule to get this property. We would still need the exceptional cases, though (like having a free-for-all `call` on any type which is Function bounded), so I think it's OK to have the current changes plus that new one about interfaces.